dtfs-central-api/api-tests/v1/tfm/deal/tfm-bss-deal.api-test.js

```diff
@@ -152,7 +154,7 @@ describe('/v1/tfm/deal/:id', () => {
               },
             })
             .to(`/v1/tfm/deals/${dealId}/snapshot`),
-        validUserTypes: ['portal'],
+        validUserTypes: ['tfm'],
```

Comment:
This is called in `trade-finance-manager-api\src\v1\controllers\deal.submit.controller.js` but it comes from portal submitting to TFM so should accept `portal` users (I don't think its every used by TFM users but might be wrong there)